src/app/components/ConnectorForm/index.tsx

```diff
@@ -42,18 +44,37 @@ function ConnectorForm({
         </div>
       ) : (
         <>
-          <div className="w-16 md:w-32 mb-8 lg:w-64 lg:mb-0">
-            <img
-              src="assets/icons/alby_logo.svg"
-              alt="Alby"
-              className="block dark:hidden w-full"
-            />
-            <img
-              src="assets/icons/alby_logo_dark.svg"
-              alt="Alby"
-              className="hidden dark:block w-full"
-            />
-          </div>
+          {image ? (
+            <>
+              <div className="w-16 md:w-32 mb-8 lg:w-[100%] lg:mb-0">
```

Comment:
You should be able to use `lg:w-full` here.

Reply:
Remove the width constraints from here to improve responsive display.

Reply:
Noted! Will work on it